spec/integration/post_show_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Show', type: :system do
+  before :each do
+    @user = User.create(name: 'Batman', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Superhero',
+                        posts_counter: 6)
+    @post = Post.create(title: 'The Dark Night', text: 'This is my first post', comments_counter: 15,
+                        likes_counter: 18, author: @user)
+    @comment1 = Comment.create(author: @user, post: @post, text: 'This is my first comment')
+    @comment2 = Comment.create(author: @user, post: @post, text: 'This is my second comment')
+    @comment3 = Comment.create(author: @user, post: @post, text: 'This is my third comment')
+    @like = Like.create(author: @user, post: @post)
+  end
+
+  scenario 'see the title of the post and who wrote it and the interactions' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content('Batman')
+    expect(page).to have_content('This is my first post')
+    expect(page).to have_content('Comments: 18')
+    expect(page).to have_content('Likes: 19')
+  end
+
+  scenario 'see the comments post' do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content('This is my first comment')
+    expect(page).to have_content('This is my second comment')
+    expect(page).to have_content('This is my third comment')
+  end
+end
```

Comment:
- [ ] Kindly add the following tests: 

``` 
Tests that asserts you can see the post body.
Tests that asserts you can see the username of each commentor.
Tests that asserts you can see the comment each commentor left.
```

- [ ] Kindly make sure that all tests written are passing ✅ 

![Screenshot_172](https://github.com/iamsjunaid/quill-quest/assets/69638013/f979cb39-ee85-489b-b0ce-a6b75417383c)


### OPTIONAL

- [ ] It would be better to separate the tests instead of grouping the tests I see you grouped tests like 
`Tests that asserts you can see how many comments it has.
 Tests that asserts you can see how many likes it has`  as interactions, you can separate each tests this will make your tests more readable 👍🏿

Reply:
- Test added 👍🏽